src/tests/WebApi.OpenAPI.SystemTest/ExtractTypeResult/TypedResultExample.cs

```diff
@@ -7,9 +7,24 @@ public TypedResultExample(string name)
         this.Name = name;
     }
 
+    public TypedResultExample(string name, OperationEnum operation)
+    {
+        this.Name = name;
+        this.Operation = operation;
+    }
+
     public string Name { get; set; }
 
     public int Count { get; set; }
 
     public string? Description { get; set; }
+
+    public OperationEnum Operation { get; set; }
+}
+
+public enum OperationEnum
```

Comment:
I see that the OpenAPI spec is properly updated with enum values as string, but I don't really understand the meaning behind each enum value's name...

Reply:
Yes the enum names were more or less very arbitrary. will change to FooBar